### PR TITLE
Dataframe

### PR DESCRIPTION
**Pull request by Moutontone**
DataFrames are now working. 

Supported features : 
- read from csv
- create from array
- print dataframe
- create dataframe from lines and column of another dataframe

Supported types :
- Integer
- String
- Float
- Boolean

### CODE REVIEW

**Comment by Polluxin** [comment]
Nice coverage.
RAS.
You just forgot to add us.

File: README.md
Comment (clergetm) [inline README.md:1]: This doc should’nt be filled because we will add here the description of the project :-1:

File: src/main/java/fr/uga/jandas/App.java
Comment (clergetm) [inline src/main/java/fr/uga/jandas/App.java:1]: Create a special demonstration main file maybe

File: src/main/java/fr/uga/jandas/Column.java [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+    String label;
+    private String type;
+    T [] elements;
+
+    public Column(String type, int nb_elements){
+        this.type = type;
+        elements = (T[]) new Object[nb_elements];
+    }
+
+    public Column(String type, int nb_elements, T [] arr){
```
Comment (clergetm) [inline src/main/java/fr/uga/jandas/Column.java:15]: So the nb_element might reduce the number of element from the original array.
## Example
`arr` is composed of **10** element and i use `Column(Integer, 2, arr)`
now the column will only be composed of the first two elements.

File: src/main/java/fr/uga/jandas/DataFrame.java [hunk 2]
```diff
@@ -0,0 +1,240 @@
...
+    Format des fichiers en entrée:
+        t0,     t1,     t2,     t3
+        l0;     l1,     l2,     l3
+        n00,    n01,    n02,    n03
+        n10,    n11,    n12,    n13
+    La première ligne donne les types des colonnes. La seconde contient les étiquettes de chaque colonne.
+    Puis chaque ligne correspond à une ligne du tableau et les virgules correspondent aux séparations
+    entre les colonnes. Les portions de texte séparées par une virgule correspondent ainsi aux contenus des
+    cellules du tableau.
+ **/
```
Comment (clergetm) [inline src/main/java/fr/uga/jandas/DataFrame.java]: French comment :-1:

File: src/main/java/fr/uga/jandas/DataFrame.java [hunk 3]
```diff
@@ -0,0 +1,240 @@
...
+    }
+
+    public String [] getLabels(){
+        return labels;
+    }
+
+    /* createFrom :
+        Create a new Dataframe based on a column set of current Dataframe.
+     */
+    public DataFrame createFrom(String [] labels){
```
Comment (clergetm) [inline src/main/java/fr/uga/jandas/DataFrame.java:149]: Bad doc :-1:

File: README.md
Comment (Moutontone) [reply README.md:1]: Agreed. The changes are removed.

File: src/main/java/fr/uga/jandas/DataFrame.java (hunk 3 above)
Comment (Moutontone) [reply src/main/java/fr/uga/jandas/DataFrame.java:149]: A proper documentation will be made in a near future. For the time being we can accept this.

File: src/main/java/fr/uga/jandas/Column.java (hunk 1 above)
Comment (Moutontone) [reply src/main/java/fr/uga/jandas/Column.java:15]: I removed this file. I will publish a demonstration file as soon as possible

**Pull request merged**